compiler/rustc_hir_typeck/src/callee.rs

```diff
@@ -948,9 +938,7 @@ impl<'a, 'tcx> DeferredCallResolution<'tcx> {
                 );
             }
             None => {
-                // This can happen if `#![no_core]` is used and the `fn/fn_mut/fn_once`
-                // lang items are not defined (issue #86238).
-                fcx.dcx().emit_err(errors::MissingFnLangItems { span: self.call_expr.span });
+                bug!("")
```

Comment:
```suggestion
                bug!("missing n/fn_mut/fn_once lang items")
```
Adding a message also makes it easier to understand why this happens when reading the code.

Reply:
Oops, I thought I had put TODO here to fill in later, but I think I just left it empty.